### ISSUE TITLE
conflicted copy loses "Conflicted copy" tag after opening it [mobile]

### ISSUE THREAD

**Issue by chaserene** [standardnotes/mobile#363]
this is the mobile equivalent of [desktop/issues/576](https://github.com/standardnotes/desktop/issues/576).

after having generated a conflicted copy, only one of the copies receive the red tag (#270), and if I open that note and then go back to the note list, the copy no longer has the tag. so basically the only indication left of the mess is that I have two notes with the same title.

3.5.15
Android 9
installed via APK from GitHub

**Comment by mobitar** [comment]
Indeed, selecting the note is our UX for removing the conflicted copy flag. Only the conflict receives the flag, and not the original. One solution could be a dedicated "conflict resolution workflow" we present for any conflicts.
